config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "projects/:id", to: "projects#show", as: :show_projects
+
+  get "contestants/", to: "contestants#index", as: :contestants
+
+  post "contestant_projects/", to: "contestant_projects#create", as: :create_contestant_projects
```

Comment:
Great RESTful routing! Just a reminder that you can include IDs in the path and that's still RESTful as well. If you didn't want to have to add a hidden field in your form for project ID, you could use this path: `"/contestants/:contestant_id/projects"` -- either are RESTful

Reply:
I think I remember asking a question on this, the action would be contestants#create, as long as there is not already a post "contestants/", to: "contestants#create", right?

Reply:
No, the action looks great and can stay the same! And to be clear, the path is also okay. I'm just sharing an alternative in case you want a path parameter in there to avoid having to pass a hidden field in the form